### DISCUSSION TITLE
Supabase and Next js Authentication.

### DISCUSSION THREAD

**Discussion by BryanGabriel-Atangana** [supabase/supabase#21857] [Questions]
How could I connect the supabase authentication to user table. I created my project with the command : npx create-next-app@latest -e with supabase. When I create users they go the the authentication part of my supabase and not the users table.

**Comment by GaryAustin1** [comment]
The most common way is to use a trigger on auth.users insert to create a row in your public users table.
https://supabase.com/docs/guides/auth/managing-user-data#using-triggers

**Comment by BryanGabriel-Atangana** [comment]
Thanks 👍🏾
